tests/Aspire.Hosting.Maui.Tests/MauiPlatformExtensionsTests.cs

```diff
@@ -649,7 +697,8 @@ private static string CreateProjectContentWithout(string excludePlatform)
 
     private static string CreateTempProjectFile(string content)
     {
-        var tempFile = Path.Combine(Path.GetTempPath(), $"test_{Guid.NewGuid()}.csproj");
+        var tempFolder = Directory.CreateTempSubdirectory();
```

Comment:
Putting the csproj directly in the temp folder would make each build take minutes if the temp folder has lots of files in it. Could also be a security concern.